spec.bs

```diff
@@ -129,6 +129,20 @@ Note: This section is non-normative.
   a direct developer-to-provider API before granting entitlements.
 </div>
 
+## History of purchases ## {#history-of-purchases}
```

Comment:
In line with the above headings, this should perhaps be "Checking Past Purchases"